root/release_group/merge.tt

```diff
@@ -31,14 +33,27 @@
                     <td>[% entity.l_type_name %]</td>
                     <td class="c">[% entity.first_release_date.year or '&#x2014;' %]</td>
                     <td>[% entity.release_count %]</td>
+                    <td >
+                      <a href="/release_group/merge?remove=[% entity.id %]&submit=remove">
+                        <button type="button" class="remove-item icon" title="[% l('Remove from merge') %]"></button>
```

Comment:
It would be more consistent to use the same full-text link (and column name) as in [aliases list](https://test.musicbrainz.org/artist/72c536dc-7137-4477-a521-567eeb840fa8/aliases) (for example).

Reply:
Hmm. That takes a lot more space, which is why I went with this, but I guess it'd work too. It just might be a bit confusing to just say "Remove" - that kinda suggests you remove the entity itself, to me?

Reply:
Maybe “Deselect”?

Reply:
TBH, I liked the icon better, and the title text "Remove from merge" is clearer and more accessible than either shorter string. "Deselect" can be confusing since the page itself is for selecting the merge target, so it sounds like it's undoing the selection.

So if consistency is an issue, I'd rather change the aliases list to use an icon too. :)

Reply:
IIRC, chhavi's mockups also heavily used icons for editing actions, e.g. small pencils. This seems smarter to me - people can process images much faster than text, and they allow adding more detailed and accessible descriptions as title text or aria attributes.

Reply:
@yvanzo: what do you think?

Reply:
> It is displayed at least 3 times, one above the other. It’s clearly undoing the selection of one element, not the whole selection.

I actually agree with you, and don't think "Deselect" would be confusing to many users — nor would "Remove." Just that neither is particularly helpful to someone using a screen reader who can't view the whole page at once.

At least if a user thinks they're purging the entity from the database by clicking the icon, they'll soon find out they didn't (and maybe we should be glad :P).

> Full-text actions are not just in the aliases list

We should make them consistent at some point, IMO, but I also think:
1. We shouldn't make a new thing worse for the sake of consistency
2. We shouldn't have to change every old thing for the sake of consistency before adding a better new thing

(i.e. I'd prefer incremental progress over having to fix everything at once, unless it's easy to do so of course :))

Reply:
Made the change back.